independent-publisher-connectors/OpenAI/apiDefinition.swagger.json

```diff
@@ -80,7 +536,7 @@
           }
         },
         "summary": "GPT3 Completes your prompt",
-        "description": "GPT3 Completes your prompt",
+        "description": "GPT3 Completes your prompt (deprecated by OpenAI - use Completion_New)",
```

Comment:
@PowerRobin please add the "deprecated" property and set it to true. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation).